articles/active-directory/cloud-infrastructure-entitlement-management/onboard-gcp.md

```diff
@@ -105,10 +107,13 @@ This option detects all projects that are accessible by the Cloud Infrastructure
 
 1. Firstly, grant Viewer and Security Reviewer role to service account created in previous step at organization, folder or project scope
 
-    To enable controller mode 'On' for any projects, add following roles to the specific projects:
-    - Role Administrators
-    - Security Admin 
-2.  Once done, the steps are listed in the screen to do configure manually in the GPC console, or programmatically with the gCloud CLI
+ > [!NOTE]
+ > To enable controller mode 'On' for any projects, add following roles to the specific projects:
+ > - Role Administrators
+ > - Security Admin 
+
+2.  Once done, the steps are listed in the screen to do configure manually in the GPC console, or programmatically via Google Cloud Shell.
+   
```

Comment:
...add these roles...

#2 is a result, not really a step. It can be a sentence under the step it belongs to. Is it a result of the Note above it? If so, it can be a sentence two returns under - Security Admin.